tutorials/introductory/usage.py

```diff
@@ -149,7 +149,7 @@
 #
 # So one can do (OO-style)
 
-x = np.linspace(0, 2, 100)
+x = np.linspace(0, 2, 100)  # This is data for plotting.
```

Comment:
```suggestion
x = np.linspace(0, 2, 100)  # sample data
```
Is this sufficient? Alternatively, "sample data for plotting"?